Jenkinsfile

```diff
@@ -47,6 +47,14 @@ pipeline {
         booleanParam(name: 'SAVE_RESULTING_NOTEBOOK', defaultValue: true,
                      description: '''Check the box to save the resulting notebooks of the run.
 Note this is another run, will double the time and no guaranty to have same error as the run from py.test.''')
+        booleanParam(name: 'TEST_MAGPIE_AUTH', defaultValue: true,
+                     description: 'Check the box to test Authentication/Authorization using Magpie/Twitcher services.')
+        // below credentials are the defaults from bootstrap script:
+        //   https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/scripts/create-magpie-authtest-user
+        password(name: 'TEST_MAGPIE_ADMIN_USERNAME', defaultValue: 'authtest',
```

Comment:
`authtest` is not an admin user ... why this is the default?

Reply:
Do you prefer I use the default admin of the server? I don't mind.
These are all configs I expect a real server to modify with their secret values.